core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -4580,7 +4580,8 @@ private FileSystemMasterAuditContext createAuditContext(String command, AlluxioU
           .setAuthType(authType)
           .setIp(ClientIpAddressInjector.getIpAddress())
           .setCommand(command).setSrcPath(srcPath).setDstPath(dstPath)
-          .setSrcInode(srcInode).setAllowed(true);
+          .setSrcInode(srcInode).setAllowed(true)
+          .setCreateTimeMillis(System.currentTimeMillis());
```

Comment:
Maybe milli second is not accurate enough to measure the cost,  how about `System.nanoTime`?

Reply:
Ok, if use System.nanoTime, I would only add useTime instead of createTime and useTime beacuse the createTime will be meaningless. And the useTime will be named mExecutionTime